### PR TITLE
Use Testcontainers Qdrant Module

### PR DESCRIPTION
**Pull request by eddumelendez**
Testcontainers 1.19.6 offers a qdrant module.

### CODE REVIEW

**Review by Anush008 — CHANGES_REQUESTED** [review]
Thank you @eddumelendez.

I have one change request.

The tests are to be run on the image version set at https://github.com/qdrant/java-client/blob/master/gradle.properties#L4-L5

Read at
https://github.com/qdrant/java-client/blob/master/src%2Ftest%2Fjava%2Fio%2Fqdrant%2Fclient%2Fcontainer%2FQdrantContainer.java#L10

File: src/test/java/io/qdrant/client/DockerImage.java [hunk 1]
```diff
@@ -0,0 +1,6 @@
+package io.qdrant.client;
+
+public class DockerImage {
+
+	public static final String QDRANT_IMAGE = "qdrant/qdrant";
```
Comment (eddumelendez) [inline src/test/java/io/qdrant/client/DockerImage.java:5]:
```suggestion
	public static final String QDRANT_IMAGE = "qdrant/qdrant:" + System.getProperty("qdrantVersion");
```

**Comment by eddumelendez** [comment]
@Anush008 thanks! I missed that one. Change was applied.

**Review by Anush008 — APPROVED** [review]
Thank you.

**Pull request merged**